frontend/src/components/common/SearchAndFilters.tsx

```diff
@@ -0,0 +1,243 @@
+          FILTER BY
+        </Text>
+        <FormControl>
+          <Grid padding="10px 0px" templateColumns="repeat(7, 1fr)" gap={7}>
+            <GridItem colSpan={2}>
+              <FormLabel fontWeight="700">Residents</FormLabel>
+              <InputGroup>
+                <InputLeftElement pointerEvents="none">
+                  <Icon />
+                </InputLeftElement>
+                <Input
+                  placeholder="Text here"
+                  value={residents}
+                  onChange={(e) => setResidents(e.target.value)}
+                />
+              </InputGroup>
+            </GridItem>
+            <GridItem colSpan={2}>
+              <FormLabel fontWeight="700">Employees</FormLabel>
+              <InputGroup>
+                <InputLeftElement pointerEvents="none">
+                  <Icon />
+                </InputLeftElement>
+                <Input
+                  value={employees}
+                  placeholder="Employee ID"
+                  onChange={(e) => setEmployees(e.target.value)}
+                />
+              </InputGroup>
+            </GridItem>
+            <GridItem colSpan={2}>
+              <FormLabel fontWeight="700">Date</FormLabel>
+              <Grid templateColumns="repeat(7, 1fr)">
+                <GridItem colSpan={3}>
+                  <InputGroup>
+                    <InputLeftElement pointerEvents="none">
+                      <Icon />
+                    </InputLeftElement>
+                    <Input
+                      value={startDate}
+                      onChange={(e) => setStartDate(e.target.value)}
+                      placeholder="YYYY-MM-DD"
+                    />
+                  </InputGroup>
+                </GridItem>
+                <GridItem
+                  colSpan={1}
+                  display="flex"
+                  justifyContent="center"
+                  alignItems="center"
+                >
+                  <Text fontSize="14px" color="#6D8788" fontWeight="700">
+                    TO
+                  </Text>
+                </GridItem>
+                <GridItem colSpan={3}>
+                  <InputGroup>
+                    <InputLeftElement pointerEvents="none">
+                      <Icon />
+                    </InputLeftElement>
+                    <Input
+                      value={endDate}
+                      onChange={(e) => setEndDate(e.target.value)}
+                      placeholder="YYYY-MM-DD"
+                    />
+                  </InputGroup>
+                </GridItem>
+              </Grid>
+            </GridItem>
+            <GridItem colSpan={1} paddingTop="32px">
+              <Button background="#FFFFFF" borderColor="#285E61" border="1px">
+                Clear all
+              </Button>
+            </GridItem>
+          </Grid>
+        </FormControl>
+      </Box>
+
+      <Accordion allowToggle marginBottom="12px">
```

Comment:
oops idk why I said to add marginBottom when in the figma the open/close filters button doesn't have a margin in the bottom, so you can remove this 😅